client/my-sites/posts/post.jsx

```diff
@@ -375,13 +280,7 @@ module.exports = React.createClass( {
 					{ this.getHeader() }
 					{ this.getPostImage() }
 					{ this.getContent() }
-					<footer className="post__info">
-						<PostRelativeTimeStatus post={ this.props.post } link={ this.getContentLinkURL() } target={ this.getContentLinkTarget() } onClick={ this.analyticsEvents.dateClick }/>
-						{
-							// Only show meta items for non-drafts
-							( this.props.post.status === 'draft' ) ? null : this.getMeta()
-						}
-					</footer>
+					<PostActions { ...{ site, post: this.props.post, toggleComments: this.toggleComments } } />
```

Comment:
I'd rather keep this depending on the `site` object instead of just passing a `siteId` and depending on `getSite` because of the work that still to be done on the comments store. There's a lot of ways to simplify the component structure providing additional selectors for comments, but I'd rather keep that on a different PR.